crates/ty_server/Cargo.toml

```diff
@@ -38,11 +38,16 @@ tracing = { workspace = true }
 tracing-subscriber = { workspace = true, features = ["chrono"] }
 
 [dev-dependencies]
+ty_server = { workspace = true, features = ["testing"] }
+
 dunce = { workspace = true }
 insta = { workspace = true, features = ["filters", "json"] }
 regex = { workspace = true }
 tempfile = { workspace = true }
 
+[features]
+testing = []
+
```

Comment:
This is still required because we conditionally add the `serde::Serialize` attribute to the `ClientOptions` and other nested structs.

Reply:
Could we use `assert_debug_snapshot` instead so that we don't need the serde serialization?

Reply:
That is not pretty ;)

Reply:
Is it? Isn't it almost the same as JSON (unless we add some manual implementations):

https://insta.rs/docs/serializers/#debug

Reply:
I would also be fine to simply always derive `Serialize` for those types. It's not that many and it simplifies the crate setup